### PR TITLE
Fixing docs Issue #1033 

### PR DESCRIPTION
**Pull request by IanoNjuguna**
Issue: #1033 

Created by @fpliger
Assigned by @madhur-tandon

### What I did

I moved the info in ```/pyscriptjs/README.md``` to ```/docs/development/developing.md```. I also pointed ```/pyscriptjs/README.md``` point to ```/docs/development/developing.md```

### CODE REVIEW

**Comment by IanoNjuguna** [comment]
Hello @FabioRosado 
I'm a first time contributor. How can I resolve the failed checks?

File: docs/development/developing.md [hunk 1]
```diff
@@ -53,6 +53,51 @@ Sometimes you might be asked to rebase main into your branch. Please refer to th
...
+			- Network:    Add `--host` to expose
+
+			────────────────── LOGS ──────────────────
+
+
+Once this is visible, open a browser at [http://localhost:8080](http://localhost:8080). This will provide a list of demos that you can run.
+
+### More information
+
+There is a forum PyScript where you can discuss the project or ask questions at [https://community.anaconda.cloud/c/pyscript](https://community.anaconda.cloud/c/pyscript)
```
Comment (FabioRosado) [inline docs/development/developing.md:100]: I know this was in the README, but I think this should be expanded to include these other places:

```
* [Discussion board](https://community.anaconda.cloud/c/tech-topics/pyscript)
* [Home Page](https://pyscript.net/)
* [Blog Post](https://engineering.anaconda.com/2022/04/welcome-pyscript.html)
* [Discord Channel](https://discord.gg/BYB2kvyFwm)
```

We have been using discord as the main place for our discussions and ideally we should include it. Probably adding these other ones would be good too 😄

**Review by FabioRosado — CHANGES_REQUESTED** [review]
Hello @IanoNjuguna  thank you for working on this issue!

I think this section would fit nicely in the `setting-up-environment.md` since the `developing` is more focus on the developing process.

The "setting up your env" touches briefly the running of `npm run dev` and this changes seem like a great addition to that section 😄 

Could you possibly move it over and reword it a bit? Thank you so much!

File: docs/development/developing.md (hunk 1 above)
Comment (IanoNjuguna) [reply docs/development/developing.md:100]: @FabioRosado I'll do it during the weekend. I've been busy but I'll make time then.

**Comment by IanoNjuguna** [comment]
@FabioRosado @fpliger I have submitted the changes you requested for review. It took me a long because I was afraid to ask questions that I did not understand, that's something I have fixed since then.

**Comment by FabioRosado** [comment]
Hello @IanoNjuguna  no worries. Thank you for getting back to it and for addressing the comments 😄 

For future reference, if you have any questions or something doesn't make sense, feel free to reach out and ask for help (either in the PR or in our discord channel) we are always happy to help 😄

**Review by FabioRosado — APPROVED** [review]

**Comment by IanoNjuguna** [comment]
Thank you @FabioRosado for offering support and assistance to me. I really feel elated to experience my first PR request in this project merged. This is a memorable experience to me.

**Comment by FabioRosado** [comment]
> Thank you @FabioRosado for offering support and assistance to me. I really feel elated to experience my first PR request in this project merged. This is a memorable experience to me.

Great to hear! It's an awesome feeling, I still remember my first PR 😄

**Comment by IanoNjuguna** [comment]
> > Thank you @FabioRosado for offering support and assistance to me. I really feel elated to experience my first PR request in this project merged. This is a memorable experience to me.
> 
> Great to hear! It's an awesome feeling, I still remember my first PR smile

I don't want to celebrate for long though. I'm ready to pick an issue that will be challenging for me, I have mastery over every Python concept other than OOP and encapsulation. Can you spare me some pointers?

**Pull request merged**